seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json

```diff
@@ -444,7 +444,8 @@
         "If no compare (callback) function is passed in, it will convert the values to strings and sort alphabetically.",
         "Here is an example of using sort with a compare function that will sort the elements from smallest to largest number:",
         "<blockquote>var array = [1, 12, 21, 2];<br>array.sort(function(a, b) {<br>&nbsp;&nbsp;return a - b;<br>});</blockquote>",
-        "Use <code>sort</code> to sort <code>array</code> from largest to smallest."
+        "Use <code>sort</code> to sort <code>array</code> from largest to smallest.",
+        "<a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort' target='_blank'>Further explanation on <code>sort</code> method</a>"
```

Comment:
@raisedadead, sorry just a markdown glitch :wink:, I fixed it now. Thanks for catching that :+1: